### ISSUE TITLE
Error:A problem occurred receiving StreamBegin from server

### ISSUE THREAD

**Issue by lomoya90** [begeekmyfriend/yasea#672]
Client receive StreamBegin(User control message) from picarto.tv server, before client send createStream command.
![image](https://user-images.githubusercontent.com/6427266/48239483-06d18380-e40a-11e8-9184-49e14460ff0c.png)
Should we do like this:
![image](https://user-images.githubusercontent.com/6427266/48239589-63cd3980-e40a-11e8-8f13-4df36b87ba6e.png)

**Comment by begeekmyfriend** [comment]
I do not know about picarto.tv server. Have you conducted sufficient test to ensure that your modification is all right for all the RTMP server?

**Comment by lomoya90** [comment]
> I do not know about picarto.tv server. Have you conducted sufficient test to ensure that your modification is all right for all the RTMP server?

Yes, I found the reason.But there is still a problem with the processing when receiving STREAM_BEGIN.
Not all servers will send STREAM_BEGIN after receiving a successful CREATE_STREAM，STREAM_BEGIN can be sent in advance.
So，nothing should be done after receiving STREAM_BEGIN.

**Comment by begeekmyfriend** [comment]
Well, you may shoot a PR for this issue.
